server/data/prisonSearchClient.test.ts

```diff
@@ -0,0 +1,45 @@
+import nock from 'nock'
+import config from '../config'
+import { PrisonerMockDataA, PrisonerMockDataB } from './localMockData/prisonSearch'
+import PrisonSearchClient from './prisonerSearchClient'
```

Comment:
seems a bit inconsistent with naming .. PrisonSearchClient, prisonerSearchClient, OFFENDER_SEARCH_API ..
Our repos tend to use _prisoner_Search

Reply:
Thanks for taking a look @steverendell. I have made an update to the PR let me know if it is consistent enough.